frontend/src/shared/helpers.tsx

```diff
@@ -752,6 +798,12 @@ export function cohortSampleFilterWhereVariables(
     ];
   } else {
     tempoWhere = [
+      {
+        billedBy_CONTAINS: parsedSearchVals[0],
+      },
+      {
+        costCenter_CONTAINS: parsedSearchVals[0],
+      },
```

Comment:
Here I'm specifying new fields that users can search by. Note that `billed` isn't included here because (1) it is a boolean field, which doesn't support suffices `_IN` and `_CONTAINS` by GraphQL, and (2) I don't expect PMs to search for specific billed value. If they do, we could consider writing a custom resolver or adding a filter/sort ability to the Billed column in the future.

Reply:
Is this something we can support purely on the frontend where a user can filter the presented table for cohorts that have not been billed yet?

Reply:
Yes. AG Grid supports this ability ([example](https://ag-grid.com/_astro/open-column-dark.551e6e75.gif)). This feature doesn't seem like a one-line change, though, so if we're moving forward with this option I think we should create a separate ticket

Reply:
Sounds good. Would you mind making that a ticket?